src/plugins/data/common/search/expressions/esaggs/request_handler.test.ts

```diff
@@ -20,12 +20,20 @@ jest.mock('../../tabify', () => ({
 }));
 
 import { tabifyAggResponse } from '../../tabify';
+import { of } from 'rxjs';
 
 describe('esaggs expression function - public', () => {
   let mockParams: MockedKeys<RequestHandlerParams>;
 
   beforeEach(() => {
     jest.clearAllMocks();
+
+    searchSourceInstanceMock.fetch$ = jest.fn().mockReturnValue(
+      of({
+        rawResponse: {},
+      })
+    );
```

Comment:
Is there a reason for setting a return value for a dummy mock instead of creating a mock function?
```suggestion
    searchSourceInstanceMock.fetch$ = jest.fn(() => of({
      rawResponse: {},
    }));
```